app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
According to the checklist, it's better to use parentheses when importing multiple modules/classes. It makes the code more readable.